diff --git a/node_modules/@monaco-editor/react/lib/cjs/Editor/Editor.js b/node_modules/@monaco-editor/react/lib/cjs/Editor/Editor.js
index 3d2e49b..baf923d 100644
--- a/node_modules/@monaco-editor/react/lib/cjs/Editor/Editor.js
+++ b/node_modules/@monaco-editor/react/lib/cjs/Editor/Editor.js
@@ -28,25 +28,25 @@ function Editor({
   path,
 
   /* === */
-  theme,
+  theme = 'light',
   line,
-  loading,
-  options,
-  overrideServices,
-  saveViewState,
-  keepCurrentModel,
+  loading = 'Loading...',
+  options = {},
+  overrideServices = {},
+  saveViewState = true,
+  keepCurrentModel = false,
 
   /* === */
-  width,
-  height,
+  width = '100%',
+  height = '100%',
   className,
-  wrapperProps,
+  wrapperProps = {},
 
   /* === */
-  beforeMount,
-  onMount,
+  beforeMount = index$3.noop,
+  onMount = index$3.noop,
   onChange,
-  onValidate
+  onValidate = index$3.noop
 }) {
   const [isEditorReady, setIsEditorReady] = React.useState(false);
   const [isMonacoMounting, setIsMonacoMounting] = React.useState(true);
@@ -221,23 +221,5 @@ Editor.propTypes = {
   onChange: PropTypes__default['default'].func,
   onValidate: PropTypes__default['default'].func
 };
-Editor.defaultProps = {
-  theme: 'light',
-  loading: 'Loading...',
-  options: {},
-  overrideServices: {},
-  saveViewState: true,
-  keepCurrentModel: false,
-
-  /* === */
-  width: '100%',
-  height: '100%',
-  wrapperProps: {},
-
-  /* === */
-  beforeMount: index$3.noop,
-  onMount: index$3.noop,
-  onValidate: index$3.noop
-};
 
 exports.default = Editor;
diff --git a/node_modules/@monaco-editor/react/lib/es/Editor/Editor.js b/node_modules/@monaco-editor/react/lib/es/Editor/Editor.js
index d36d1e6..8d22b54 100644
--- a/node_modules/@monaco-editor/react/lib/es/Editor/Editor.js
+++ b/node_modules/@monaco-editor/react/lib/es/Editor/Editor.js
@@ -18,25 +18,25 @@ function Editor({
   path,
 
   /* === */
-  theme,
+  theme = 'light',
   line,
-  loading,
-  options,
-  overrideServices,
-  saveViewState,
-  keepCurrentModel,
+  loading = 'Loading...',
+  options = {},
+  overrideServices = {},
+  saveViewState = true,
+  keepCurrentModel = false,
 
   /* === */
-  width,
-  height,
+  width = '100%',
+  height = '100%',
   className,
-  wrapperProps,
+  wrapperProps = {},
 
   /* === */
-  beforeMount,
-  onMount,
+  beforeMount = noop,
+  onMount = noop,
   onChange,
-  onValidate
+  onValidate = noop
 }) {
   const [isEditorReady, setIsEditorReady] = useState(false);
   const [isMonacoMounting, setIsMonacoMounting] = useState(true);
@@ -211,23 +211,5 @@ Editor.propTypes = {
   onChange: PropTypes.func,
   onValidate: PropTypes.func
 };
-Editor.defaultProps = {
-  theme: 'light',
-  loading: 'Loading...',
-  options: {},
-  overrideServices: {},
-  saveViewState: true,
-  keepCurrentModel: false,
-
-  /* === */
-  width: '100%',
-  height: '100%',
-  wrapperProps: {},
-
-  /* === */
-  beforeMount: noop,
-  onMount: noop,
-  onValidate: noop
-};
 
 export default Editor;
